app/views/outcomes-report/PageOutcomesReport.vue

```diff
@@ -191,53 +201,66 @@ export default {
       }
       this.setStats({ stats, includeSubOrgs, kind })
     },
 
     setStats ({ stats, includeSubOrgs, kind }) {
       if (!stats) return
+      this.org = this.setOrgAndSubOrgs(stats)
+      if (this.includeOther) {
+        this.otherOrg = this.setOrgAndSubOrgs(stats, 'other')
+      }
+    },
+
+    setOrgAndSubOrgs (stats, product = 'current') {
       let subOrgs = []
-      if (includeSubOrgs) {
-        for (const childKind of orgKinds[kind].childKinds) {
-          subOrgs = subOrgs.concat(stats.current[childKind + 's'] || [])
+      if (this.includeSubOrgs) {
+        for (const childKind of orgKinds[this.kind].childKinds) {
+          subOrgs = subOrgs.concat(stats[product][childKind + 's'] || [])
         }
         for (const [index, subOrg] of subOrgs.entries()) {
           subOrg.initiallyIncluded = Boolean(!subOrg.archived && index < this.subOrgLimit && subOrg.progress && subOrg.progress.programs && (subOrgs.length > 1 || this.subOrgLimit === 1))
           // TODO: better way to get rid of redundant info if there is only one subOrg
         }
       }
-      this.subOrgs = Object.freeze(subOrgs) // Don't add reactivity
+      if (product === 'current') {
+        this.subOrgs = Object.freeze(subOrgs) // Don't add reactivity
+      } else {
+        this.otherSubOrgs = Object.freeze(subOrgs) // Don't add reactivity
+      }
 
-      const licenses = stats.current.licenses
-      const orgs = stats.current[kind + 's']
+      const licenses = stats[product].licenses
+      const orgs = stats[product][this.kind + 's']
       if (orgs) {
         orgs[0].subOrgs = this.subOrgs
         orgs[0].newLicenses = licenses
-        this.org = Object.freeze(orgs[0]) // Don't add reactivity
-        console.log('   ... got our org', this.org)
+        const org = Object.freeze(orgs[0]) // Don't add reactivity
+        console.log('   ... got our org', org)
+        return org
       }
     },
 
     async fetchUsingBackgroundJob ({ kind, orgIdOrSlug, includeSubOrgs, country, startDate, endDate, includeOther }) {
-      const otherProduct = includeOther ? this.otherProduct : undefined
-      const resp = await createJob(JOB_TYPE, { kind, orgIdOrSlug, includeSubOrgs, country, dateRange: { startDate, endDate } }, otherProduct)
+      const resp = await createJob(JOB_TYPE, { kind, orgIdOrSlug, includeSubOrgs, country, dateRange: { startDate, endDate } })
+      let resp2
+      if (includeOther) {
+        resp2 = await createJob(JOB_TYPE, { kind, orgIdOrSlug, includeSubOrgs, country, dateRange: { startDate, endDate } }, this.otherProduct)
+      }
       const jobId = resp?.job
+      const jobId2 = resp2?.job
 
       if (!jobId) {
         console.log('failed to create job', jobId, resp)
         this.loadingText = $.i18n.t('loading_error.could_not_load')
         return
       }
-      const stats = await this.pollJob(jobId)
+      const stats = await this.pollJob(jobId, jobId2)
```

Comment:
so instead of polling in parallel, you can just poll 1 first and then second. It would be even simpler and anyway both jobs would have started in backend so mostly less polling calls. But this is an enhancement so can do later